### PR TITLE
Divide queries file into entities files

### PR DESCRIPTION
**Pull request by uyen3893**
Please review it

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 const express = require('express')
 const bodyParser = require('body-parser')
-const db = require('./queries')
+const queryUsers = require('./queriesusers')
```
Comment (phkhuong) [inline index.js]: please correct the file name.

File: queriesusers.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
+const Pool = require('pg').Pool
```
Comment (phkhuong) [inline queriesusers.js]: please remove Pool reference because we don't need it anymore

File: querybookchecks.js [hunk 3]
```diff
@@ -0,0 +1,202 @@
+const Pool = require('pg').Pool
```
Comment (phkhuong) [inline querybookchecks.js]: please remove Pool reference because we don't need it anymore

File: querybooks.js [hunk 4]
```diff
@@ -0,0 +1,203 @@
+const Pool = require('pg').Pool
```
Comment (phkhuong) [inline querybooks.js]: please remove Pool reference because we don't need it anymore

File: querycategories.js [hunk 5]
```diff
@@ -0,0 +1,202 @@
+const Pool = require('pg').Pool
```
Comment (phkhuong) [inline querycategories.js]: please remove Pool reference because we don't need it anymore

File: queryreports.js [hunk 6]
```diff
@@ -0,0 +1,167 @@
+const Pool = require('pg').Pool
```
Comment (phkhuong) [inline queryreports.js]: please remove Pool reference because we don't need it anymore

**Review by phkhuong — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (uyen3893) [reply index.js]: I fixed it

File: queriesusers.js (hunk 2 above)
Comment (uyen3893) [reply queriesusers.js]: I fixed it

File: querybooks.js (hunk 4 above)
Comment (uyen3893) [reply querybooks.js]: I fixed it

File: querybookchecks.js (hunk 3 above)
Comment (uyen3893) [reply querybookchecks.js]: I fixed it

File: querycategories.js (hunk 5 above)
Comment (uyen3893) [reply querycategories.js]: I fixed it

File: queryreports.js (hunk 6 above)
Comment (uyen3893) [reply queryreports.js]: I fixed it

**Review by phkhuong — APPROVED** [review]

**Pull request merged**